### ISSUE TITLE
Suggestions for MacOS version

### ISSUE THREAD

**Issue by biozz** [ivanesmantovich/xkbswitch.nvim#1]
Hi! Thank you for making NeoVim alternative of the original xkbswitch. Love to see people making more and more lua stuff out there.

I tried your plugin, and it didn't work for me, so I decided to give you a couple of suggestions.

When you build `vovkasm/input-source-switcher` on MacOS you get `libInputSourceSwitcher.dylib` and `issw`. That `issw` binary is basically `xkb-switch`. It also has `-l` option, which produces the following on my machine:

```
com.apple.keylayout.ABC
com.apple.keylayout.Russian
```

There was also a change on recent MacOS versions on Apple Silicon Macs, which changed permissions for `/usr/local/lib` and `/usr/local/bin`. I don't want to use sudo, so I put libs and bins into another directory. I think people do the same, but I am not sure. Anyway, it would be great to have an ability to change default paths.

In short, my suggestions are:

- add ability to use `issw`
- improve US layout finder
- add ability to change default paths

**Comment by ivanesmantovich** [comment]
Thank you for the first issue!
I will improve the layout finder on mac and add the ability to manually pass the path to the .lib file as soon as i recover from the surgery i had today, it should not take longer than 3-4 days.

**Comment by biozz** [comment]
I hope you are getting better. Meanwhile, I submitted a proof of concept which works on my Mac.

**Comment by Emptyfruit** [comment]
Hi ) Thank you for your work.
I guess the layout finder on mac  needs some more improvement, on my mac the US layput has a name 
`com.apple.keylayout.US`.

And i get the following error: 
```
Error detected while processing /Users/Emptyfruit/.config/nvim/after/plugin/xkb-switch.lua:
E5113: Error while calling lua chunk: .../site/pack/packer/start/xkbswitch.nvim/lua/xkbswitch.lua:52: (xkbswitch.lua) Error occured: could not
find the English layout. Check your layout list. (xkb-switch -l / issw -l / g3kb-switch -l)
stack traceback:
```
